src/sentry/integrations/jira_server/client.py

```diff
@@ -144,8 +147,18 @@ def get_project_key_for_id(self, project_id) -> str:
     def get_versions(self, project):
         return self.get_cached(self.VERSIONS_URL % project)
 
-    def get_priorities(self):
-        return self.get_cached(self.PRIORITIES_URL)
+    def get_priorities(self, project: str):
+        """
+        Fetches all priorities associated with the project. Jira Server does not provide a
+        single API to fetch priorities for a project, so we instead fetch the project's
```

Comment:
It's so weird 😅 I assume everyone using this API for their UI has to do the same.